configs/Kinetics/C2D_8x8_R101_2_GPUs_NO_SLURM.yaml

```diff
@@ -0,0 +1,58 @@
+TRAIN:
```

Comment:
We can follow the conventions of the folder. The filename could simply be 2D_8x8_R101_2_GPUs.yaml

We can add _SLURM for configs that use slurm.